UnityProject/Assets/Scripts/HealthV2/Living/CirculatorySystem/BodyPartBlood.cs

```diff
@@ -84,13 +84,8 @@ public partial class BodyPart
 		/// </summary>
 		public float BloodThroughput => bloodThroughput;
 
-		public enum RequiredBlood
-		{
-			Large = 100,
-			Medium = 20,
-			Low = 10,
-			Minuscule = 1
-		}
+
+		public float currentBloodSaturation = 0;
```

Comment:
private variable?

Reply:
is needed for organ functionality

Reply:
then captalize the first C or make it private and make a public getter